developer-docs-site/docs/guides/move-guides/nft-tutorial/Move.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "NFT-tutorial"
```

Comment:
Why is this called the NFT tutorial? Bit confused here.

I think Mayaan forked the Move tutorial into the NFT tutorial, but this is a different story? Do we have that forked NFT tutorial already somewhere?

Reply:
I copied this over verbatim from the private repo. Did you mean to ask @0xmaayan ?

Reply:
Which private repo? This is about porting the generic, public Move tutorial in the move-language repo. 

I do not know what @0xmaayan did here before, but the tutorial I have in mind is to augment the Move book, and it should not be specific to NFTs.

Reply:
Actually, i recall I heard about this from @0xchloe .

Reply:
I think there is misunderstanding here. 
I worked on a different tutorial that was added to the aptos-core repo because of some future plans, see https://github.com/aptos-labs/aptos-core/pull/6381 

I don't know this tutorial.

Reply:
Sorry, @0xmaayan . I tried to correct above and note @0xchloe brought this content to light. Here is the private repo:
https://github.com/aptos-labs/nft-tutorial

I tried to not put that in a public issue or pull request. But we need some way to track all of this private stuff. I will attempt to find the Slack thread where this was discussed and copy relevant parties from here to there.